contrib/fwupd.spec.in

```diff
@@ -77,6 +77,7 @@ BuildRequires: libarchive-devel
 BuildRequires: libcbor-devel
 BuildRequires: gobject-introspection-devel
 BuildRequires: gcab
+BuildRequires: pandoc
```

Comment:
i guess we need this for debian packaging too

Reply:
isnt that the magic `<control />` thing?